### ISSUE TITLE
Add Kleros arbitration as an option to SafeSnap plugin

### ISSUE THREAD

**Issue by auryn-macmillan** [snapshot-labs/snapshot#1913]
If a Reality Module has Kleros set as its oracle, we should expose UI elements to:
1. request arbitration
2. display the status of arbitration
3. call any public functions that keep the process rolling along

**Comment by bonustrack** [comment]
Closing this issue, it was transferred on snapshot-labs/snapshot

**Comment by zzuziak** [comment]
Hey @mktcode, what's the status of this issue?

**Comment by mktcode** [comment]
I agree. Those controls should be added to the SafeSnap UI. In this context I was actually wondering for some time now how this plays together with the new UMA implementation and the overall design of everything. After I've put quite some time into refactoring the code, I feel like there's some significant amount of overlap/duplication.
